jormungandr-scenario-tests/src/main.rs

```diff
@@ -134,6 +134,71 @@ pub fn scenario_1(mut context: Context<ChaChaRng>) {
     controller.finalize();
 }
 
+pub fn scenario_point_to_point(mut context: Context<ChaChaRng>) {
+    let scenario_settings = prepare_scenario! {
+        "P2P",
+        &mut context,
+        topology [
+            "Leader4",
+            "Leader3" -> "Leader4",
+            "Leader2" -> "Leader3",
+            "Leader1" -> "Leader4",
+        ]
+        blockchain {
+            consensus = GenesisPraos,
+            number_of_slots_per_epoch = 60,
+            slot_duration = 1,
+            leaders = [ "Leader2" ],
+            initials = [
+                account "unassigned1" with   500_000_000,
+                account "delegated1" with  2_000_000_000 delegates to "Leader1",
+            ],
+        }
+    };
+
+    let mut controller = scenario_settings.build(context).unwrap();
+
+    controller.monitor_nodes();
+    let leader4 = controller.spawn_node("Leader4", true).unwrap();
```

Comment:
It would be nice for manual repro if the enumeration followed the order in which the nodes should be launched. I had to find out from code which one was the "seed" node.

Reply:
One thing you can do is add the `--document` to the command line of the scenario. It will create a `.dot` file that you can convert into png with graphviz. The file is in the test directory when you ran with `--root-dir`.

there is still a million things to write on this tool. We have each command available and in order available in the controller. we just need to dump it in a script file to reproduce the tests.